superchain/configs/sepolia/boba.yaml

```diff
@@ -0,0 +1,22 @@
+name: boba
+chain_id: 28882
+public_rpc: https://sepolia.boba.network
+sequencer_rpc: https://sepolia.boba.network
+explorer: https://testnet.bobascan.com
+
```

Comment:
Did you rerun the lastest `add-chain.sh` on this PR? I'm expecting to see a `superchain_level` in this file.

You can see this [PR](https://github.com/ethereum-optimism/superchain-registry/pull/140/files#diff-61f8eef010d99a9c901eb9b47102af2e4bf4f1e96dc0eb003fe420cfeed85a0c). It takes the level, determined if you're adding a standard or frontier chain, and adds it to this yaml file.

Reply:
Yes, I will upgrade contracts next to fix unit tests.